src/test/kotlin/no/skatteetaten/aurora/gobo/integration/spotless/SpotlessCnameServiceReactiveTest.kt

```diff
@@ -0,0 +1,28 @@
+package no.skatteetaten.aurora.gobo.integration.spotless
+
+import assertk.assertThat
+import assertk.assertions.hasSize
+import assertk.assertions.isEqualTo
+import no.skatteetaten.aurora.gobo.CnameAzureBuilder
+import no.skatteetaten.aurora.mockmvc.extensions.mockwebserver.executeBlocking
+import no.skatteetaten.aurora.mockmvc.extensions.mockwebserver.url
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.jupiter.api.Test
+import org.springframework.web.reactive.function.client.WebClient
+
+class SpotlessCnameServiceReactiveTest {
```

Comment:
Har du sjekket om spotless genererer kontrakt stubber? Mokey og cantus gjør det hvertfall. Burde vi spille inn at de bør ha det evt.

Reply:
Hva tenker du @jarlehansen 
Gir det nok verdi med stubbene?

Reply:
Hvis spotless genererer det så bør vi bruke det ja, gir enda bedre feedback på at man faktisk gjør det rette kallet, gir riktig response

Reply:
Det blir ikke generert stubber for spotless, om dere mener vi bør ha det kan jeg høre om det kan legges til generering i spotless